core/src/main/scala/com/gu/tableversions/core/model.scala

```diff
@@ -100,3 +100,10 @@ final case class TableDefinition(name: TableName, location: URI, partitionSchema
 sealed trait TableVersion
 final case class PartitionedTableVersion(partitionVersions: Map[Partition, Version]) extends TableVersion
 final case class SnapshotTableVersion(version: Version) extends TableVersion
+
+case class FileFormat(name: String) extends AnyVal
```

Comment:
Do you think there'd be any value in locking this down to a closed set of known-supported formats?

Reply:
Not really, I don't think it makes sense for this library to make assumptions as to which set of formats that client code will be using.